### PR TITLE
add two popular building values

### PR DESCRIPTION
**Pull request by matkoniecz**
https://taginfo.openstreetmap.org/tags/building=outbuilding#overview

https://taginfo.openstreetmap.org/tags/building=allotment_house#overview

building=outbuilding is popular, but this is fueled by imports from PRG in Poland, not an organic growth (at least as of 2022-07), so this unspecific value is unsearchable

@tyrasd Would you be interested in listing of popular unsupported values for say `shop` `amenity` `barrier` `leisure` `natural` (with adjusted thresholds, lower than 100k used for buildings)?

I believe that JOSM has similar script that is listing popular tags (keys?) and they maintain list of popular tags which are deliberately not supported.

### CODE REVIEW

File: data/presets/building/allotment_house.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+        "{building}"
+    ],
+    "geometry": [
+        "area"
+    ],
+    "tags": {
+        "building": "allotment_house"
+    },
+    "matchScore": 0.5,
+    "name": "Allotment house"
```
Comment (tyrasd) [inline data/presets/building/allotment_house.json:13]:
```suggestion
    "name": "Allotment House"
```

File: data/presets/building/_outbuilding.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    "icon": "maki-building",
+    "fields": [
+        "{building}"
+    ],
```
Comment (tyrasd) [inline data/presets/building/_outbuilding.json:5]: This is not necessary, as the fields will be inherited from the parent preset by default.

```suggestion
```

File: data/presets/building/allotment_house.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "icon": "maki-building",
+    "fields": [
+        "{building}"
+    ],
```
Comment (tyrasd) [inline data/presets/building/allotment_house.json:5]:
```suggestion
```

**Review by tyrasd — CHANGES_REQUESTED** [review]
some minor tweaks (see below)

File: data/presets/building/allotment_house.json (hunk 3 above)
Comment (tyrasd) [reply data/presets/building/allotment_house.json]: Do you remember which one did you use as a template? I checked a few of the existing building presets and didn't spot this issue.

Comment (matkoniecz) [reply data/presets/building/allotment_house.json]: `allotment_house.json` and `outbuilding.json` - several others have this field and extra ones - it is likely necessary to repeat `{building}` when one adds extra fields, right?

(checked with `rg --context 3 "\{building\}"` using ripgrep)

Comment (tyrasd) [reply data/presets/building/allotment_house.json]: correct: when you want to add extra fields to a sub-preset the `"fields": ["{parent}", "extra"]` syntax is the way to go. But to just get the same fields in the sub-preset, it is not necessary to explicitly include the parent.

**Comment by tyrasd** [comment]
//cc @1ec5 do you think these names are acceptable in en-US?

**Review by tyrasd — APPROVED** [review]

**Pull request merged**